site/usage/compile.md

```diff
@@ -20,14 +20,6 @@ If you want access to the compiled Vega spec from a Javascript program, you can
 var vgSpec = vl.compile(vlSpec).spec;
 ```
 
-Then, you can continue to use the [Vega runtime](https://github.com/vega/vega/wiki/Runtime)'s `vg.parse.spec` method to render your Vega spec.
```

Comment:
1) `vg.parse.spec` method is deprecated. (I think it is replaced by `vega.parse`.)
2) I thought we should promote to use Vega-Embed to render the compiled spec too.
3) I felt that this info is not related to *Compile* and not necessary to be here.
So I just decided to delete this incorrect info.